### PR TITLE
chore: upgrade to node 12.13

### PR DESCRIPTION
**Pull request by wardpeet**
## Description

Upgraded all packages to node 12 and bumped all peerdeps

Using the following config in Lerna to keep minors below 1
```
versions.set('gatsby-admin', '0.10.0-next.0');
versions.set('gatsby-plugin-graphql-config', '0.9.0-next.0');
versions.set('gatsby-recipes', '0.11.0-next.0');
versions.set('gatsby-theme', '0.10.0-next.0');
```

### CODE REVIEW

File: integration-tests/cache-resilience/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "inspect-state": "node utils/inspect-state"
   },
   "dependencies": {
-    "gatsby": "^2.19.30",
+    "gatsby": "^3.0.0",
```
Comment (wardpeet) [inline integration-tests/cache-resilience/package.json:16]: should be bumped to next

**Review by vladar — DISMISSED** [review]

**Comment by thibaudcolas** [comment]
Just commenting to say I’m surprised this was marked as a `chore` in conventional-commits terminology, and as such this change wasn’t surfaced [in the gatsby-cli CHANGELOG](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-cli/CHANGELOG.md), where I would expect the information to be. I would have expected changing the supported Node version to be a breaking change – particularly since gatsby-cli’s recommended install method is `npm install -g gatsby-cli`. 

For my project upgrading from Node 10 to Node 12 isn’t an immediate option, so it’s pretty important to know what version of gatsby-cli to install that will work with it.

**Comment by wardpeet** [comment]
Hey,

We bumped node 10 - 12 during a major. At least it should have 😬 Don't you get an error when doing it on a wrong ndoe version?

**Comment by thibaudcolas** [comment]
I do, but there is no mention of when the version bump happened that I could see (and the error message doesn't say either), so I resorted to trying all of the minor releases one after the other until I reached v2.

**Pull request closed without merging**